<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>-1</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuDerniers_fichiers_ouverts">
     <property name="title">
      <string>Derniers fichiers ouverts</string>
     </property>
     <addaction name="actionmachin_txt"/>
     <addaction name="actiontruc_fic"/>
     <addaction name="actionmon_machin_txt"/>
    </widget>
    <addaction name="actionOuvrir"/>
    <addaction name="actionQuitter"/>
    <addaction name="menuDerniers_fichiers_ouverts"/>
    <addaction name="actionSauvegarder"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action_propos"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionPolice"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>Ã€ propos</string>
   </property>
  </action>
  <action name="actionmachin_txt">
   <property name="text">
    <string>machin.txt</string>
   </property>
  </action>
  <action name="actiontruc_fic">
   <property name="text">
    <string>truc.fic</string>
   </property>
  </action>
  <action name="actionmon_machin_txt">
   <property name="text">
    <string>mon machin .txt</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionPolice">
   <property name="text">
    <string>Police</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
